src/com/Communication.hpp

```diff
@@ -93,11 +95,13 @@ class Communication {
    *
    * @param[in] acceptorName Name of calling participant.
    * @param[in] requesterName Name of remote participant to connect to.
+   * @param[in] meshName Name of the involved mesh
    * @param[in] acceptorRank Rank of accepting server, usually the rank of the current process.
    * @param[in] requesterCommunicatorSize Size of the requester (N)
    */
   virtual void acceptConnectionAsServer(std::string const &acceptorName,
                                         std::string const &requesterName,
+                                        std::string const &meshName,
```

Comment:
Maybe, instead of `meshName`, we could use a more neutral name. In the end, from a design perspective, there is no mesh needed to establish a communication and the `meshName` only serves as a tag for establishing the communication. What about `std::string const tag`?

Reply:
True, I initially thought about calling it `context`, which didn't quite fit. `tag` is perfect!